TypeScriptReact.YAML-tmLanguage

```diff
@@ -115,6 +115,7 @@ repository:
         '2': { name: entity.name.tag.tsx }
         '3': { name: support.class.component.tsx }
       end: (?=[/]?>)
+      contentName: meta.tag.attributes.tsx
```

Comment:
Are you sure that you also want to include #jsx-tag-attributes-illegal and #comment that gets applied inside this scope to have meta.tag.attributes.tsx ?

Reply:
Yes, my intent was the have everything after the tag name but before the closing `>` scoped with  `meta.tag.attributes.tsx`